CMakeLists.txt

```diff
@@ -1,13 +1,26 @@
-cmake_minimum_required(VERSION 2.8.3)
+################################################################################
+# Set minimum required version of cmake, project name and compile options
+################################################################################
+cmake_minimum_required(VERSION 3.5)
 project(aws_robomaker_small_warehouse_world)
 
-find_package(catkin REQUIRED COMPONENTS
-  gazebo_ros
-)
-
-catkin_package()
+################################################################################
+# Find ament packages and libraries for ament and system dependencies
+################################################################################
+find_package(ament_cmake REQUIRED)
+find_package(gazebo_ros REQUIRED)
 
-install(DIRECTORY launch models worlds maps rviz
-  DESTINATION ${CATKIN_PACKAGE_SHARE_DESTINATION}
+################################################################################
+# Install
+################################################################################
+install(
+  DIRECTORY launch models worlds maps rviz
+  DESTINATION share/${PROJECT_NAME}
 )
 
+ament_environment_hooks("${CMAKE_CURRENT_SOURCE_DIR}/env-hooks/aws_robomaker_small_warehouse_world.dsv.in")
+
```

Comment:
Add export dependencies for downstream packages using this world
```
ament_export_dependencies(ament_cmake)
ament_export_dependencies(gazebo_ros)
```

Reply:
I have added them